### ISSUE TITLE
g8os.alpha: logs should go to redis per corex sub process

### ISSUE THREAD

**Issue by despiegk** [g8os/core0#14]
- is stdout/err
- the logs self rotate if more than X lines (through log stored procedure in redis)

**Comment by iwanbk** [comment]
@zaibon @muhamadazmy 

There are 3 PRs regarding this:

- https://github.com/g8os/core.base/pull/3

Implementation of redis logger as library in core.base

- https://github.com/g8os/core0/pull/18

Update core0 to use this new logger library

- https://github.com/g8os/coreX/pull/3

Update coreX to use this new logger library

Because of vendoring with godep, The last 2 PRs still need to be updated after the first PR getting merged.

Notes about this new logger:

1. We will lost logs that sent before redis server started.
The library can buffer the logs until redis started
2. logs copying from coreX's redis to core0's redis still not optimal, one by one.

I'm not sure if we need to handle two cases above in this stage.
But i can  fix #1 after i updated go-raml docs.

**Comment by iwanbk** [comment]
```
We will lost logs that sent before redis server started.
The library can buffer the logs until redis started
```
This issue has been fixed
